src/MESONT/angle_mesocnt.cpp

```diff
@@ -0,0 +1,392 @@
+/* ----------------------------------------------------------------------
```

Comment:
Since you are adding new styles in a package, those files must be added to `src/.gitignore` so that git will ignore the copy when using the traditional build system.